.github/workflows/sync-and-validate-master.yml

```diff
@@ -0,0 +1,25 @@
+name: Sync and validate master branch on Arm64 Windows
+
+on:
+  pull_request:
+    branches: [ "dev" ]
+  workflow_dispatch:
+  schedule:
+    - cron: "0 3 * * *"
+
+jobs:
+  sync-fork:
+    name: Sync master
+    runs-on: ubuntu-latest
+
+    steps:
+    - env:
```

Comment:
I am not sure if it's mix of spaces and tabs but the indentation seems to be weird here.

Reply:
a good catch. Fixed